airflow/api_connexion/openapi/v1.yaml

```diff
@@ -604,6 +604,40 @@ paths:
         '404':
           $ref: '#/components/responses/NotFound'
 
+  /dags/{dag_id}/dagRuns/{dag_run_id}/state:
+    parameters:
+      - $ref: '#/components/parameters/DAGID'
+      - $ref: '#/components/parameters/DAGRunID'
+
+    post:
+      summary: Set a state of DAG run
+      description: Set a state of DAG run
+      x-openapi-router-controller: airflow.api_connexion.endpoints.dag_run_endpoint
+      operationId: update_dag_run_state
+      tags: [UpdateDagRunState]
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/UpdateDagRunState'
```

Comment:
Should we accept DAGRun here similar to `PATCH /dags/{dag_id}`?
https://github.com/apache/airflow/blob/fb11ba5c42c2547e6550480eca1c5db14ea76f99/airflow/api_connexion/openapi/v1.yaml#L466

Reply:
It will give room to updating the state to any value, and the state on DAGRun is readonly, making it accept a value will lead to setting the DAGrun to running in the API